client-sdk/go/modules/consensusaccounts/consensus_accounts.go

```diff
@@ -11,6 +11,13 @@ import (
 	"github.com/oasisprotocol/oasis-sdk/client-sdk/go/types"
 )
 
+var (
+	// PendingWithdrawalAddress is the address of the internal pending withdrawal account in the consensus_accounts module.
+	PendingWithdrawalAddress = types.NewAddressForModule("consensus_accounts", []byte("pending-withdrawal"))
```

Comment:
`"consensus_accounts"` could be moved to a constant (same goes for `accounts` in accounts,go).

Reply:
There already are these module constants. Should just use them https://github.com/oasisprotocol/oasis-sdk/blob/8b71d83f7d5eb0091674c013b5c25b1267157e13/client-sdk/go/modules/consensusaccounts/types.go#L152